app/amity.py

```diff
@@ -0,0 +1,309 @@
+import random
+import os
+
+from app.livingspace import LivingSpace
+from app.office import Office
+from models.amity_database import Room, Person as PersonModel, Session
+from app.people import Person
+
+
+session = Session().create_session()
```

Comment:
Remove this line. When initialising `Amity` you are passing the session instance eg `Amity(session)` as allocating it to `self.session`